examples/patterns/pagination.html

```diff
@@ -3,14 +3,83 @@
 title: Pagination
 category: _patterns
 ---
+<h4>Standard</h4>
```

Comment:
Can we use the titles too, please? It's nicer to just have the mark up of the pattern to copy out.